core/message-handler.ts

```diff
@@ -1,15 +1,18 @@
-// import toast from "react-hot-toast";
+import toast from "react-hot-toast";
+import { t } from "@/resources/utils/translate";
 
 class MessageHandler {
   handleError(error: string): null {
     console.error("Error occurred:", error);
-    // toast.error(error || "An unexpected error occurred");
+
+    toast.error(t(error || "An unexpected error occurred"));
```

Comment:
I dislike this way of having a "A normal text" as parameter of the i18n.

Poeple thing is better because if something goes wrong at least properly e English is shown, and not and `id`

But if tomorrow we need to change the value, we need to change the `id` as well.
Or leave the all english words to be replaced for other which is confusing


EN "The Dog" : "The Dog"
ES "The Dog" : "El Perro"

Now I want to put the cat

EN "The Cat" : "The Cat"
ES "The Cat" : "El Gato"
🙅 I had to modify the key


EN "The dog" : "The Cat"
ES "The Dog" : "El Gato"
🙅 I don't modify they key, but now is confusing the code and misleading